src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars--0">
```

Comment:
по бем-у модифікатор має бути разом з блоком starts, не в середині цього блоку, а з ним.
перший div тут не обов'язковий 
```suggestion
      <div class="starts stars--0">
```